release/_hotfix.sh

```diff
@@ -176,10 +176,51 @@ function compare_jars {
 			fi
 		fi
 
-		if [ -n "${jar_descriptions}" ]
+		local new_jar_descriptions=""
+
+		if (echo "${jar_descriptions}" | grep -q ".class")
+		then
+			mkdir -p "${_BUILD_DIR}/tmp/jar1" "${_BUILD_DIR}/tmp/jar2"
+
+			while IFS= read -r line
+			do
+				if (echo "${line}" | grep -q ".class")
+				then
+					unzip -p "${jar1}" "${line}" > "${_BUILD_DIR}/tmp/jar1/${line##*/}"
+					unzip -p "${jar2}" "${line}" > "${_BUILD_DIR}/tmp/jar2/${line##*/}"
```

Comment:
@lucasmiranda0, can you please use the `$(basename "${line}")` instead of `${line##*/}`?

In addition, I believe you can add this to a variable since we're using more than one time. :wink: